spec/player_spec.rb

```diff
@@ -72,7 +72,24 @@
 
   describe "#highest_word_score" do
     it "returns the highest scoring word score" do
-      expect(@player2.highest_word_score).to eq 8 
+      expect(@player2.highest_word_score).to eq 8
+    end
+  end
+
+  describe "#tiles" do
+    it "returns the collection of letters that the player can play" do
+      expect(@player5.tiles).to be_an_instance_of Array
+    end
+    it "contains a maximum of 7 tiles" do
+      expect(@player5.tiles.length).to be <= 7
```

Comment:
Could you explain the reason for this particular test, and how you've chosen to write it?